app/eventsv1_test.go

```diff
@@ -94,7 +94,7 @@ func TestVerifiableEventsV1EventsFromData(t *testing.T) {
 							0x5b, 0x5d, 0x7d,
 						}, // serialized bytes
 					),
-					1, // mmr index
+					18446744073709551614, // mmr index
```

Comment:
Oh wait, why is this -1 ?

Reply:
no real reason, just wanted something big enough that we're sure we're in the realm of uint64 without actually landing on the boundary